spec/models/validators/utaac_decision_validator_spec.rb

```diff
@@ -10,16 +10,16 @@
       title: double,
       summary: double,
       body: "body",
-      tribunal_decision_category: category,
+      tribunal_decision_categories: categories,
       tribunal_decision_decision_date: "2015-11-01",
       tribunal_decision_judges: [double],
-      tribunal_decision_sub_category: sub_category,
+      tribunal_decision_sub_categories: sub_categories,
     )
   }
   let(:document_type) { "utaac_decision" }
 
   subject(:validatable) { UtaacDecisionValidator.new(entity) }
 
-  it_behaves_like "tribunal decision sub_category validator"
+  include_examples "tribunal decision sub_categories optional"
```

Comment:
Added shared examples testing validation passes when `tribunal_decision_sub_categories` are not present. This tests the new behaviour.